src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -250,10 +249,13 @@ public function iConfirmMyOrder(): void
      * @When I select :shippingMethod shipping method
      * @When /^the (?:visitor|customer) proceed with ("[^"]+" shipping method)$/
      * @Given /^the (?:visitor|customer) has proceeded ("[^"]+" shipping method)$/
-     * @When /^the visitor try to proceed with ("[^"]+" shipping method) in the customer cart$/
      */
     public function iProceededWithShippingMethod(ShippingMethodInterface $shippingMethod): void
     {
+        $y = $this->getCart();
```

Comment:
Hmm, i suggest using more descriptive names of variables :D
```suggestion
        $token = $this->getCart();
```
i guess it is a `token` value :)